tiledb/sm/filter/filter_pipeline.h

```diff
@@ -323,7 +323,7 @@ class FilterPipeline {
   std::vector<shared_ptr<Filter>> filters_;
 
   /** The max chunk size allowed within tiles. */
-  uint32_t max_chunk_size_;
+  uint32_t max_chunk_size_{0};
```

Comment:
Looks like this class might need some C.41 / capi treatment. I see we still use `set_max_chunk_size`. @bekadavis9 what do you think?